metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py

```diff
@@ -18,254 +28,1381 @@
+        schema_metadata = get_schema_metadata(
+            self.report,
+            dataset_name,
+            self.platform,
+            columns,
+            schema_fields,
+        )
 
-    """
+        dataset_snapshot.aspects.append(schema_metadata)
+        db_name = self.get_db_name(inspector)
 
-    __visit_name__ = "TIMESTAMP"
+        yield from self.add_table_to_schema_container(dataset_urn, db_name, schema)
+        mce = MetadataChangeEvent(proposedSnapshot=dataset_snapshot)
+        wu = SqlWorkUnit(id=dataset_name, mce=mce)
+        self.report.report_workunit(wu)
+        yield wu
+        dpi_aspect = self.get_dataplatform_instance_aspect(dataset_urn=dataset_urn)
+        if dpi_aspect:
+            yield dpi_aspect
+        subtypes_aspect = MetadataWorkUnit(
+            id=f"{dataset_name}-subtypes",
+            mcp=MetadataChangeProposalWrapper(
+                entityType="dataset",
+                changeType=ChangeTypeClass.UPSERT,
+                entityUrn=dataset_urn,
+                aspectName="subTypes",
+                aspect=SubTypesClass(typeNames=["ML Models"]),
+            ),
+        )
+        self.report.report_workunit(subtypes_aspect)
+        yield subtypes_aspect
+        yield from self._get_domain_wu(
+            dataset_name=dataset_name,
+            entity_urn=dataset_urn,
+            entity_type="dataset",
+            sql_config=sql_config,
+        )
 
-    def __init__(self, timezone=False, precision=None):
-        """Construct a new :class:`_types.TIMESTAMP_WITH_PRECISION`.
+    def get_model_properties(
+        self, inspector: Inspector, schema: str, model: str
+    ) -> Tuple[Optional[str], Dict[str, str], Optional[str]]:
+        """
+        Returns ml models related metadata information to show in properties tab
+            eg. ml model attribute and ml model specification information.
 
-        :param timezone: boolean.  Indicates that the TIMESTAMP type should
-         enable timezone support, if available on the target database.
-         On a per-dialect basis is similar to "TIMESTAMP WITH TIMEZONE".
-         If the target database does not support timezones, this flag is
-         ignored.
-        :param precision: integer.  Indicates the PRECISION field when provided
+        Args:
+            inspector (Inspector): inspector obj from reflection engine
+            schema (str): schema name
+            model (str): ml model name
+        Returns:
+            Tuple[Optional[str], Dict[str, str], Optional[str]]
+        """
+        description: Optional[str] = None
+        properties: Dict[str, str] = {}
+        # The location cannot be fetched generically, but subclasses may override
+        # this method and provide a location.
+        location: Optional[str] = None
+        try:
+            # SQLAlchemy stubs are incomplete and missing this method.
+            # PR: https://github.com/dropbox/sqlalchemy-stubs/pull/223.
```

Comment:
seems like these comments were copied without checking them

Reply:
thanks for pointing it out , extra comments have been removed in current commit